app/static/src/app/types/responseTypes.ts

```diff
@@ -51,7 +51,7 @@ export interface OdinParameter {
 }
 
 export interface OdinModelResponseError {
-    line: string[],
+    line: string[] | null,
```

Comment:
two things - I've fixed odin.api here to not return null, per the schema https://github.com/mrc-ide/odin.api/pull/14

second, this should surely be `number[]` not `string[]`?